services/spd/opteed/opteed_pm.c

```diff
@@ -66,6 +66,9 @@ static void opteed_cpu_suspend_handler(u_register_t max_off_pwrlvl)
 	assert(optee_vector_table);
 	assert(get_optee_pstate(optee_ctx->state) == OPTEE_PSTATE_ON);
 
+	write_ctx_reg(get_gpregs_ctx(&optee_ctx->cpu_ctx), CTX_GPREG_X0,
+		max_off_pwrlvl);
```

Comment:
Please align with opening `(`.

Reply:
sure will add the spaces. btw as a reference I used the code from line 104...should I fix that as well on the same commit?

Reply:
Just this is fine.